### PR TITLE
Adding quote from greg about lesson pilots

### LINKED ISSUE #180: Suggestion to incorporate a quote from Greg Wilson in Operations
I like this quote by Greg Wilson:
> "No lesson survives first contact with learners"

from https://third-bit.com/talks/writing-book/#19 that could go nicely at: https://carpentries.github.io/lesson-development-training/operations.html#pilot-workshops.

### PR DESCRIPTION
**Pull request by sstevens2**
Fixes #180

Let me know if you are okay with the way I linked the slide @anenadic found with this quote.

### CODE REVIEW

File: episodes/operations.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ over to another styling when the lesson moves e.g. into The Carpentries Lab.
 
 ## Pilot Workshops
 
+::::::::::::::::::::::::::::::::::::::::::::::: testimonial
+"No lesson survives first contact with learners".
+
+--- [Greg Wilson](https://third-bit.com/talks/writing-book/#19)
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
```
Comment (tobyhodges) [inline episodes/operations.md:80]:
```suggestion
::::::::::::::::::::::::::::::::::::::::::::::: testimonial

"No lesson survives first contact with learners".

--- [Greg Wilson](https://third-bit.com/talks/writing-book/#19)

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
```

**Review by tobyhodges — APPROVED** [review]
The slide link is a further improvement, I think. 

I have suggested one very minor change, to keep the fenced div syntax [consistent with the `sandpaper` style guide](https://carpentries.github.io/sandpaper-docs/style.html#whitespace).

Thanks, @sstevens2!

**Comment by tobyhodges** [comment]
Good catch! I'll have to work back through that lesson to ensure the styling guidelines are applied...

**Comment by sstevens2** [comment]
Making a quick PR now to the sandpaper docs

**Pull request merged**